.examples/nginx/db.env

```diff
@@ -0,0 +1,3 @@
+MYSQL_PASSWORD=
```

Comment:
I don't know if you can comment in .env files, if possible give the link to the docker instance env doc. I think the password is mandatory unless you specify no password env variable for mariadb. maybe you can put a todo step in a README.md or put some warning that will throw an error unless fixed in the .env file ?

Reply:
I'm not aware of a way to use comments in env files.